UI/bugs/profile.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="StyleSheet" type="text/css" href="../Css/style.css">
+    <title>Profile</title>  
+</head>
+<body class="picture-container">
+    <h2>Profile Picture</h2>
+        <div class="imgcontainer">
+            <img src="../Images/AVATAr.png" width="120px" height="140px" alt="Avatar" class="avatar">
+        
+        <address class="Adress">
+            Name:devdenk<br>
+            Email:dev@gmail.com<br>
+            Joined:02/10/2019
+        </address>
+        <div>
+            <p> 7 Entries created since joining <br>3Entries currently active</p>
+        </div>
+        </div>
+        <div class = profile-btn>
+                <button type="reset" class="prof-btn1"><a href="#">Upload Profile</a></button>

```

Comment:
What do you think of changing the blue color of your buttons to white? would be better?

Reply:
but may be the problem that might arise is that i have already used white color as background when hovered

Reply:
Or i can make white color buttons and bluecolor when hovered?would be better?